src/types.ts

```diff
@@ -206,8 +206,6 @@ export enum ContentLanguage {
 
 export type I18nRecord = Partial<Record<ContentLanguage, string>>;
```

Comment:
Leaving the `ContentLanguage` enum for now, it's not really used anymore other than a type.